pycytominer/cyto_utils/cp_aggregate.py

```diff
@@ -0,0 +1,138 @@
+"""
+Functions for aggregating "non-compartment" features
```

Comment:
I think `cp_image_features.py` is a more descriptive name. 

Something to use the word `image` at least, as all these functions deal with processing them.

Reply:
And then I think you need to define in the docstring that you're using the specific `image` features from the CellProfiler output